other-docs/getting-started/differences-from-wp.md

```diff
@@ -0,0 +1,86 @@
+---
+order: 100
+---
+# For WordPress Developers
+
+If you're an experienced WordPress developer, you'll feel right at home with Altis.
+
+We've designed Altis to be the best of WordPress, with extra functionality and tools to make it even easier to work on projects. Most of your existing processes, tools, plugins, and themes will work with no changes.
+
+
+## Your project structure
+
+Unlike traditional WordPress projects, your Altis project's codebase will look a little different. (Altis projects are based off a [WordPress Skeleton pattern](https://github.com/markjaquith/WordPress-Skeleton).)
+
+Your project will contain a `content/` directory, which works just like `wp-content/ that you may be used to already. This can contain your plugins and themes; Altis is compatible with most existing WordPress plugins and themes.
```

Comment:
```suggestion
Your project will contain a `content/` directory, which works just like `wp-content/` that you may be used to already. This can contain your plugins and themes; Altis is compatible with most existing WordPress plugins and themes.
```